README.md

```diff
@@ -1,226 +1,188 @@
-![Kin Logo](kin.png)  ![Stelalr Logo](stellar.png)
+![Kin Logo](kin.png)
+
+# KIN Python SDK for Kin Blockchain
 
-# KIN Python SDK for Stellar Blockchain
```

Comment:
why removing the badges?

Reply:
Tests are not complete so I want to finish them and then add the badges back